internal/commands/app/create.go

```diff
@@ -66,23 +67,34 @@ func (cmd *CommandCreate) Handler(profile *cli.Profile, ui terminal.UI, clients
 		return err
 	}
 
-	var ds dataSource
-	if cmd.inputs.DataSource != "" {
-		ds, err = cmd.inputs.resolveDataSource(clients.Atlas, groupID)
+	var cs clusterService
+	if cmd.inputs.Cluster != "" {
+		cs, err = cmd.inputs.resolveCluster(clients.Atlas, groupID)
+		if err != nil {
+			return err
+		}
+	}
+
+	var ds dataLakeService
+	if cmd.inputs.DataLake != "" {
+		ds, err = cmd.inputs.resolveDataLake(clients.Atlas, groupID)
```

Comment:
[megaminor q] I assume we are good linking both a cluster and data lake an app if someone provides both? I really like that tbh just wanted to make sure

Reply:
intentional behavior here, no reason we'd have to limit the user to either/or (come to think of it, we probably could support these as an array, but we'll leave that for a future release)